src/client/components/Nav/index.js

```diff
@@ -20,6 +20,8 @@ const Nav = ({
     });
   };
 
+  // useEffect(addAnalytics('research'));

```

Comment:
Ignore linting error here: this useEffect needs to be added eventually, but I'm getting errors when I include it. Will investigate.